websites/A/AniMi Club/metadata.json

```diff
@@ -0,0 +1,47 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "546262814424170498",
+		"name": "official_riktik"
+	},
+	"service": "AniMi Club",
+	"description": {
+		"en": "Watch anime online for free. Large database of the best anime with russian voiceover in good quality.",
+		"ru": "Смотреть аниме онлайн бесплатно. Большая база лучших аниме с русской озвучкой в хорошем качестве.",
+		"uk_UA": "Дивитися аніме онлайн безкоштовно. Велика база найкращих аніме з російською озвучкою в хорошій якості."
+	},
+	"url": [
+		"animi.club",
+		"www.animi.club",
+		"dev.animi.club"
+	],
+	"version": "1.0.0",
+	"logo": "https://i.imgur.com/BZtDh0g.png",
+	"thumbnail": "https://i.imgur.com/Anc197L.png",
+	"color": "#e11d48",
+	"category": "anime",
+	"tags": [
+		"anime",
+		"аниме",
+		"animi",
+		"аними",
+		"animiclub",
+		"анимиклаб"
+	],
+	"iframe": true,
+	"iFrameRegExp": ".*",
```

Comment:
If possible, set the iframe regex to the urls that it accesses

Reply:
I'm a little confused about what you're talking about 😅

Reply:
The current value `.*` is alright, but means that it runs on any iframes on the page. We'd prefer it if the regex only matches the iframe urls that the page actually uses for its video features.

For example, if the page uses video sources from `example.com` and `foo.com`, a potential regex could be:
```json
"(example[.]com|foo[.]com)[/]"
```

Reply:
Unfortunately there are links in the iframe that I may not be aware of. The API I use for the player may give different links.